saleor/core/management/commands/populatedb.py

```diff
@@ -85,6 +86,11 @@ def sequence_reset(self):
             cursor.execute(commands.getvalue())
 
     def handle(self, *args, **options):
+        settings.PLUGINS = [
```

Comment:
Wondering if this is not a workaround. Maybe we should try setting some fake address in site settings that is required by Avalara.

Reply:
I am not sure that we want to have each new order created by populated DB directly send to Avalara.

Reply:
Probably not, are we sending them to Avalara now?

Reply:
Yes

Reply:
Avalara as a plugin doesn't know if the order comes from populateDB or from normal user.